source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/CustomActionNewType.mm

```diff
@@ -0,0 +1,111 @@
+//
+//  CustomActionNewType
+//  CustomActionNewType.mm
+//
+//  Copyright © 2019 Microsoft. All rights reserved.
+//
+
+#import "CustomActionNewType.h"
+#import <AdaptiveCards/ACOBaseActionElementPrivate.h>
+#import <AdaptiveCards/SharedAdaptiveCard.h>
+
+@implementation CustomActionNewType
+
+-(instancetype)init
+{
+    self = [super init];
+    return self;
+}
```

Comment:
You can leave this out and just let the parent's `init` method be inherited implicitly.

Reply:
thanks

---
In reply to: [252495022](https://github.com/Microsoft/AdaptiveCards/pull/2325#discussion_r252495022) [](ancestors = 252495022)